packages/cli/src/s3-store.ts

```diff
@@ -185,6 +189,8 @@ class S3KVStore implements IKVStore {
   }
 
   get(key: string): Promise<any> {
+    Metrics.count(LOAD_S3_QUEUE_ADD, 1)
+    Metrics.observe(LOAD_S3_QUEUE_SIZE, this.#loadingLimit.size)
```

Comment:
This is a `get` call, should we move these lines to the `put` call? I don't think the S3 store is used anymore, but adding these doesn't hurt.

Reply:
well - the way i read this, for whatever reason the .get of the s3 storage is adding a leveldb get call into the queue.  so for monitoring the queue, i want it here.  

The put doesn't add to the queue, it just does the put synchronously it seems